Create samuel-singer.md

Cache123 opened the pull request:
Sorry I thought I sent this over correctly, realized I hadn't sent it correctly

robbiejaeger commented:
@Cache123 Looks like the in `1708/mod-4/homework-submissions.md` file you added some extra space on lines 9 and 11. I know it's minor, but it's good practice to make sure you don't edit lines that you don't intend to.

Update your PR without the spaces, and then I can merge this. Thanks!

Cache123 commented:
@robbiejaeger I updated the spacing change that I accidentally created on lines 9 and 11! Also added my House of Vars Blog Post link as well. Thanks for pointing that out!
